index.html

```diff
@@ -831,7 +830,7 @@ <h3>Claims</h3>
             src="diagrams/claim-example.svg" alt="Pat has an alumniOf
             property whose value is Example University">
           <figcaption style="text-align: center;">
-A basic claim expressing that Pat is an alumni of "Example University".
+A basic claim expressing that Pat is an alumnus of "Example University".
```

Comment:
```suggestion
A basic claim expressing that Pat is an alum of "Example University".
```

Was trying to keep it gender neutral, missed the mark w/ alumni, fixing now with the gender neutral form (I had no idea alumni, alumnus, alumnae, alumns, had so many gender forms).

Reply:
Raises a question about `alumniOf` as the related property name.